airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination_async/partial_messages/PartialAirbyteRecordMessage.java

```diff
@@ -14,6 +15,10 @@ public class PartialAirbyteRecordMessage {
   @JsonProperty("stream")
   private String stream;
 
+  @JsonProperty("emitted_at")
```

Comment:
did we double check emitted at is correctly deserialised and passed through?

Reply:
Yup, I checked on the Postgres connection cause I know the `emitted_at` field for DB connections is fixed, if it was still using the `Instant.now()` then it would be a changing value